### ISSUE TITLE
Desmume to iNDS transfer

### ISSUE THREAD

**Issue by theninjasushi** [WilliamLCobb/iNDS#272]
Hi everyone, been playing Pokemon HeartGold for a while on Desmume (v9.11), and I recently got iNDS (v1.9.1) on my iPad, and want to transfer the save file Desmume -> iNDS. I tried going through dropbox sync and by manually dragging in save files into the apps section on iTunes, but no luck. I'm able to open HeartGold on my iPad, but it doesn't seem to find my save file, just defaults to starting a new game, despite putting the .sav and .dsv files into the "battery" folder on iNDS.

TLDR; Is there any way to transfer my saved game from Desmume to iNDS?

Any advice would be appreciated!
